alpine/resin

```diff
@@ -0,0 +1,66 @@
+#!/sbin/openrc-run
```

Comment:
@petrosagg found the manpage http://manpages.ubuntu.com/manpages/wily/man8/openrc-run.8.html. However, it doesn't work as I expect. Service status will always be starting (because we run a shell script) even when I force it into background, which is not good.